checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
The test case was mentioned in #880, without a reference to the jtreg test. So I added it as an explicit test.
Do you want me to change the "Issue" to "Pull request"?